### PR TITLE
Outlook

### PR DESCRIPTION
**Pull request by tik9**
Be back..;)

### CODE REVIEW

File: outlook/outlook-quiz.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+
```
Comment (Ebazhanov) [inline outlook/outlook-quiz.md:1]: please add the name of the topic, the example you can find in the other quiz

File: outlook/outlook-quiz.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+#### 1. What can you set a reminder for in Outlook?
+- [] all of these answers
+- [] contacts
+- [] email messages
+- [] appointments and meetings
```
Comment (Ebazhanov) [inline outlook/outlook-quiz.md:7]:
```suggestion
- [ ] all of these answers
- [ ] contacts
- [ ] email messages
- [ ] appointments and meetings
```

File: outlook/outlook-quiz.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+
+#### 2. What is the default permission that other people in your organization have to your calendar?
+- [] none of these answers
+- [] They can edit.
+- [] They can view all details.
+- [] They can view titles and locations.
```
Comment (Ebazhanov) [inline outlook/outlook-quiz.md:14]:
```suggestion
- [ ] none of these answers
- [ ] They can edit.
- [ ] They can view all details.
- [ ] They can view titles and locations.
```

**Review by Ebazhanov — COMMENTED** [review]
Great job @tik9! Thank you but please revert back `.huskyrc` and `.prettierrc` (you can read more about husky [here](https://www.npmjs.com/package/husky)) I am using it every time when I do push!

plus left some comments

**Review by tik9 — COMMENTED** [review]
Just did the changes you requested;)

**Pull request merged**